laadunseuranta/clj-src/harja_laadunseuranta/tarkastusreittimuunnin/tarkastusreittimuunnin.clj

```diff
@@ -42,11 +42,26 @@
   [nykyinen-reittimerkinta seuraava-reittimerkinta]
   (let [jatkuvat-havainnot-pysyvat-samana? (= (:jatkuvat-havainnot nykyinen-reittimerkinta)
                                               (:jatkuvat-havainnot seuraava-reittimerkinta))
-        seuraava-piste-samalla-tiella? (boolean (or
-                                                  (nil? (:tr-osoite nykyinen-reittimerkinta))
-                                                  (nil? (:tr-osoite seuraava-reittimerkinta))
```

Comment:
Mikäli tapahtui ajallinen gäppi ja seuraava piste oli nil, niin se otettiin seuraavan reitin ensimmäiseksi pisteeksi. Tästä eteenpäin tämä ehto oli aina true, koska edellisessä merkinnässä ei ole tietä, mutta seuraavassa on. Jatkossa tällaisessa tilanteessa reitti katkaistaan.